airbyte-integrations/connectors/source-sonar-cloud/source_sonar_cloud/spec.yaml

```diff
@@ -3,12 +3,22 @@ connectionSpecification:
   $schema: http://json-schema.org/draft-07/schema#
   title: Sonar Cloud Spec
   type: object
-  required:
-    - user_token
-    - organization
-    - component_keys
+  oneOf:
+    - required:
+      - user_token
+      - organization
+      - component_keys
+    - required:
+      - user_token
+      - qualifiers
+      - component_keys
   additionalProperties: true
   properties:
+    api_url:
+      type: string
+      title: Sonar API URL
+      description: Either SonarCloud or SonarQube API URL.
+      default: https://sonarcloud.io/api
```

Comment:
Can you explain why you need a configurable URL?

Reply:
SonarQube is the open-source edition of the SonarCloud which company can selfhost.